pkg/report/sarif.go

```diff
@@ -197,16 +206,28 @@ func toPathUri(input string) string {
 	return strings.ReplaceAll(input, "\\", "/")
 }
 
-func sarifSeverityLevel(severity string) string {
+func getSeverityLevel(vuln *types.Vulnerability) string {
+	// If there is CVSS from NVD, take this one.
+	// else take the first existed CVSS
+	if nvd, ok := vuln.CVSS[vulnerability.NVD]; ok {
+		return fmt.Sprintf("%.1f", nvd.V3Score)
+	}
+	for _, cvss := range vuln.CVSS {
+		return fmt.Sprintf("%.1f", cvss.V3Score)
+	}
```

Comment:
We can use `SeveritySource` to take the vendor CVSS score. I feel we should not use the NVD score. It is possible that the severity is Low and the score is 9.5. The score depends on the severity source for consistency.

The priority is
1. Vendor score (it would be ok if it is NVD)
2. Mapping score